src/panels/lovelace/cards/types.ts

```diff
@@ -361,6 +362,7 @@ export interface PictureEntityCardConfig extends LovelaceCardConfig {
   show_name?: boolean;
   show_state?: boolean;
   theme?: string;
+  alt_text?: string;
```

Comment:
I think for the picture entity card we should build the `alt` text based on the rest of the configuration (e.g. using the entity friendly name and state).  Whether or not the picture changes with state, camera is included, or the entity name is shown in text would all affect the rendered template.  Maybe put this one aside for a future PR?

Reply:
Hmm, yeah, I agree. I will revert this subset of changes.